src/blocks/notification/Notification.tsx

```diff
@@ -1,91 +1,12 @@
 import { FC } from 'react';
 import styled from 'styled-components';
-import { Cross } from '../icons';
 import { NotificationProps } from './Notification.types';
 import { toast } from 'sonner';
-import { getTextVariantStyles } from 'blocks/Blocks.utils';
 
-const NotificationContainer = styled.div`
```

Comment:
We won't be removing this from here. but instead add an overlay prop which will allow us to render custom component.